System.IO.Packaging/ContentType.cs

```diff
@@ -43,13 +43,14 @@
 // 
 //-----------------------------------------------------------------------------
 
-using System;
-using System.Collections.Generic;   // For Dictionary<string, string>
-using System.Text;                  // For StringBuilder
-using System.Diagnostics;           // For Debug.Assert
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
 
 namespace System.IO.Packaging
 {
+    using Properties;
```

Comment:
The
```
using Properties;
```
statement (with `Properties` being shorthand for `System.IO.Packaging.Properties`) must be positioned within the 
```
namespace System.IO.Packaging
``` 
declaration to make the compiler find the `System.IO.Packaging.Properties.Resources` class before it finds the `System.Resources` namespace. If we put a
```
using System.IO.Packaging.Properties;
```
statement outside of the namespace declaration (together with the other `using` statements), `Resources` would be interpreted as the `System.Resources` *namespace* rather than the `System.IO.Packaging.Properties.Resources` *class*.